frontend/src/app/projects/models/diagrams/model-diagram-dialog/model-diagram-dialog.component.ts

```diff
@@ -73,6 +85,11 @@ export class ModelDiagramDialogComponent {
       });
   }
 
+  async ngOnInit() {
+    const data = await this.getUser();
+    this.cachedUsername = data.name;
+  }
```

Comment:
I think, we no longer need this here when adjusting the functionality in here and changing the backend route to use `get_own_user` (i.e., we no longer need the id of the current user)

Reply:
good catch! Yes we do not need it anymore but not due to the change in the backend but rather due to the change of using the `UserService` to get the username